src/website/src/pages/ec/components/list/docs/usage.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  Lists group together related pieces of information so they are clearly
+  associated with each other and easy to read. Lists are good from a structural
```

Comment:
```suggestion
  A list is usually organized vertically and it consists of a single continuous column of items displayed in rows. The content of each item can be plain text, an external link or a short text paragraph. Even if lists can be nested inside of each other (to display sub-sections for example) it's not recommended nesting lists deeper than three levels in order to avoid confusion.
```